pandas/conftest.py

```diff
@@ -170,3 +170,67 @@ def string_dtype(request):
     * 'U'
     """
     return request.param
+
+
+@pytest.fixture(params=["float16", "float32", "float64"])
+def float_dtype(request):
```

Comment:
I was also asked to add these fixtures in #21432.  I think it might be best to merge #21432 first to make backporting easier since it's tagged for 0.23.2?  Not sure if it really matters though.

Your code looks 95% identical to mine, so I'll just modify mine to match yours prior to pushing so there shouldn't be any significant issues regardless of which PR gets merged first.  The only potential sources of differences:
- I was asked to not include float16 since it's barely supported (https://github.com/pandas-dev/pandas/pull/21432#discussion_r194702410)
- Looks like the existing convention is to not include a blank line between the docstring and `return`

Reply:
@jschendel : I'll address all of your questions:

* The merging doesn't particularly matter at this point.  I would hold on modifying your fixtures for the moment.  Let's ask @jreback in your PR.

* `float16`, ah that's true.  I'll remove that.

* I generally prefer a newline after the docstring.  Facilitates reading IMO.